recipes/acado/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+import os
+import glob
+
+from conans import ConanFile, CMake, tools
+
+
+class AcadoConan(ConanFile):
+    name = "acado"
+    description = "ACADO Toolkit is a software environment and algorithm collection for automatic control and dynamic optimization."
+    license = "LGPL-3.0"
+    topics = ("conan", "acado", "control", "optimization", "mpc")
+    homepage = "https://github.com/acado/acado"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "cmake/qpoases.cmake"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "examples": [True, False],
+        "testing": [True, False],
```

Comment:
These can be removed in CCI, consumer just need the lib and headers

Reply:
done